db/migrate/20170309125642_add_alert_ems_ref.rb

```diff
@@ -0,0 +1,6 @@
+class AddAlertEmsRef < ActiveRecord::Migration[5.0]
+  def change
+    add_column :miq_alert_statuses, :event_ems_ref, :string
+    add_index :miq_alert_statuses, :event_ems_ref
```

Comment:
Is this ems_ref related to the manageiq events table at all?  If not, then this name might be confusing.